app/views/state_file/questions/tax_refund/_bank_details.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="white-group">
   <p class="spacing-below-0"><%= t(".bank_title") %></p>
   <p class="text--small"><%= t(".foreign_accounts") %></p>
-  <div class="form-group-tight">
+  <div>
```

Comment:
just out of curiosity why was this removed?

Reply:
it makes the bullet points vertical which is more aesthetic